src/utilities/hopscotch_hash.h

```diff
@@ -47,13 +47,13 @@
 #include <string.h>
 #include <stdio.h>
 #include <limits.h>
-#include <math.h>
+//#include <math.h>
 
 #ifdef HYPRE_USING_OPENMP
 #include <omp.h>
 #endif
 
-#include "_hypre_utilities.h"
```

Comment:
But this looks strange. `_hypre_utilities.h` should contain this file, and this file includes `_hypre_utilities.h` back? Maybe I missed the point here?

Reply:
@rfalgout do you have comments on this line? Thanks!

Reply:
It doesn't look necessary and probably should be deleted.  Is that the question you have?

Reply:
Yes, just to confirm it's OK to delete. Thanks @rfalgout .